addon/session-stores/adaptive.js

```diff
@@ -104,10 +104,11 @@ export default Base.extend({
 
     @method persist
     @param {Object} data The data to persist
+    @return {Ember.RSVP.Promise} The promise object persisting the data in the store.
```

Comment:
I think saying "promise object" might lead to people confusing this with a `DS.PromiseObject`. Maybe "A promise that resolves when the data was succesfully persisted and rejects otherise." would be better.